### PR TITLE
Fix Shopify #207

### PR DESCRIPTION
**Pull request by jslawler**

### CODE REVIEW

File: source/trackers.json [hunk 1]
```diff
@@ -859,6 +865,17 @@
...
+        "shop.app": "shopify",
+        "shopifyapps.com": "shopify",
+        "shopifycdn.net": "shopify",
+        "shopify.com.au": "shopify",
+        "shopify.com.mx": "shopify",
+        "shopify.co.za": "shopify",
+        "shopify.dev": "shopify",
+        "shopifynetwork.com": "shopify",
+        "shopifypreview.com": "shopify",
+        "shopifysvc.com": "shopify",
```
Comment (jellizaveta) [inline source/trackers.json:878]: `"shopifysvc.com"` looks like tracker or analytics. maybe we can rewrite `"shopify_stats"` with category 6 - site_analytics (+ `stats.shopify.com`, `v.shopify.com`)

**Review by jellizaveta — CHANGES_REQUESTED** [review]

**Review by Alex-302 — APPROVED** [review]

File: source/trackers.json (hunk 1 above)
Comment (bos37) [reply source/trackers.json]: > `"shopifysvc.com"` looks like tracker or analytics. maybe we can rewrite `"shopify_stats"` with category 6 - site_analytics (+ `stats.shopify.com`, `v.shopify.com`)

**Review by bos37 — CHANGES_REQUESTED** [review]
> ~~****~~

**Comment by bos37** [comment]
> > ~~****~~

**Review by bos37 — APPROVED** [review]
>

**Review by Plentywolf — CHANGES_REQUESTED** [review]
Looks  great

**Pull request merged**